README.md

```diff
@@ -44,14 +57,16 @@ You can find multiple examples at
 
 ### Arguments
 
+You can pass additional arguments to the `cypress/run` job to override any default behaviors.
```

Comment:
The args section is tough to take in and decipher as-is. Is there a way to get these into a table or something so it's easier to grok at a glance?